src/components/HomeView.jsx

```diff
@@ -32,24 +40,40 @@ const styles = (theme) => ({
     paddingTop: theme.spacing(4),
   },
   title: {
-    color: theme.palette.primary.main,
-    fontSize: '54px',
-    lineHeight: '66px',
-    paddingBottom: theme.spacing(8),
+    color: theme.palette.secondary.dark,
+    fontSize: '1.5rem',
+    fontWeight: 700,
+    paddingBottom: '1rem',
   },
-  wrapper: {
-    display: 'flex',
+  tabsRoot: {
+    color: '#333F52',
     fontFamily: theme.typography.fontFamily,
-    justifyContent: 'center',
-    padding: theme.spacing(4),
-    margin: theme.spacing(4),
+    height: 18,
+    fontSize: '1rem',
+    fontWeight: 600,
+    lineHeight: 18,
+    textAlign: 'center',
   },
-  width: {
-    ...theme.mixins.containerWidth,
+  tabSelected: {
+    fontWeight: 700,
+    color: theme.palette.secondary.dark,
+    bottomBar: '6px',
+  },
+  tabsIndicator: {
+    borderBottom: `6px solid ${theme.palette.secondary.main}`,
+    transition: 'none',
   },
 });
 
 class HomeView extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      tabValue: '/datasets',
```

Comment:
I'm not sure that we want to use component state...shouldn't we just be using the redux store.  Component state should, in my opinion, be used exceedingly rarely

Reply:
Can't we get that from the router value from the global store?